### DISCUSSION TITLE
content/cherokee-language-fonts/

### DISCUSSION THREAD

**Discussion by giscus** [CherokeeLanguage/CherokeeLessons.com#17] [General]
# content/cherokee-language-fonts/

https://www.cherokeelessons.com/content/cherokee-language-fonts/

**Comment by ChungDTien** [comment]
Hi Joseph. It's Chung. Love your typeface. I was wondering if I can use your Unicode platform sometime in the future to add my new design in it. I've used Chris Harvey's Unicode plate form because I got his permission but I like to try a simpler Unicode platform. I like your permission to use your Unicode layout and replace it with my experimental Cherokee fonts if that's ok. Email me if you have time. I'm not in a rush. This is just something I'm thinking of doing. chungdehtien@gmail.com.

**Comment by michael-conrad** [comment]
I don't think Joseph visits this site very often. The contact form at [Joseph Erb Contact Form](https://www.josepherb.com/contact) would probably be a better way to contact Joseph.

-Magali

**Comment by ChungDTien** [comment]
Thank you. I have his link on my LinkedIn page. It's nothing that urgent.
Thank you for your time. Chung
